[SharedUX] SCSS migration kibana_overview plugin

paulinashakirova opened the pull request:
## Summary

This PR is a part of SCSS migration of SharedUX team code.
Here is a [meta](https://github.com/elastic/kibana-team/issues/1417) issue for it.

Commit:
[SharedUX] SCSS migration kibana_overview plugin

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/287088)
* Commit: 8bc83dd506556aeff40b450b5bfd44cb68626b64

### Failed CI Steps
* [Jest Tests #14](https://buildkite.com/elastic/kibana-pull-request/builds/287088#0195ceb0-e767-44e4-a399-9d5e18046222)
* [Jest Tests #14](https://buildkite.com/elastic/kibana-pull-request/builds/287088#0195ced5-0008-4b56-8612-9183ec2e5acf)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/287088#0195ceb0-e767-44e4-a399-9d5e18046222) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/287088/jobs/0195ceb0-e767-44e4-a399-9d5e18046222/artifacts/0195ced4-eb6f-4f6c-a3a2-7defcab2aad4) Jest Tests #<span></span>14 / ManageData render
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/287088#0195ced5-0008-4b56-8612-9183ec2e5acf) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/287088/jobs/0195ced5-0008-4b56-8612-9183ec2e5acf/artifacts/0195ced9-0846-40a5-8ad8-f25a31063b33) Jest Tests #<span></span>14 / ManageData render

cc @paulinashakirova

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287088","number":287088,"commit":"8bc83dd506556aeff40b450b5bfd44cb68626b64"}],"number":287088}
buildkite-pr-comment-->

Commit:
update snapshot

paulinashakirova commented on src/platform/plugins/private/kibana_overview/public/components/overview/overview.tsx:
```diff
@@ -354,11 +406,13 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
-                }`}
+                css={css({
+                  flexDirection: 'column',
+                  '&, & > *': {
```
I could not see any difference with 
marginBottom: '0 !important',

Uploading Screen Recording 2025-03-25 at 22.12.15.mov…


marginTop: '0 !important',

eokoneyo replied:
typically one tries to stay away from using `!important`,  and instead provide a declaration with a high specificity than what we are looking to override.

paulinashakirova replied:
Yes, I agree that `!important` is not a bit "code smell". I removed it since it makes no difference - I don't think we need it.

Commit:
styles

elasticmachine commented:
Pinging @elastic/appex-sharedux (Team:SharedUX)

Commit:
Merge branch 'main' into scss-to-js-kubana-overview

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/287150)
* Commit: cb279e42f073d9071988b15c9cab750e5da42728

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/1d457e4b1bdc444f956f7f8feec6cc3415a4a605) | [after](https://github.com/elastic/kibana/commit/cb279e42f073d9071988b15c9cab750e5da42728) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 93 | 80 | -13 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/1d457e4b1bdc444f956f7f8feec6cc3415a4a605) | [after](https://github.com/elastic/kibana/commit/cb279e42f073d9071988b15c9cab750e5da42728) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 47.8KB | 37.9KB | -9.9KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/1d457e4b1bdc444f956f7f8feec6cc3415a4a605) | [after](https://github.com/elastic/kibana/commit/cb279e42f073d9071988b15c9cab750e5da42728) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 5.7KB | 5.6KB | -47.0B |



### History
* :broken_heart: [Build #287088](https://buildkite.com/elastic/kibana-pull-request/builds/287088) failed 8bc83dd506556aeff40b450b5bfd44cb68626b64

cc @paulinashakirova

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287150","number":287150,"commit":"cb279e42f073d9071988b15c9cab750e5da42728"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287088","number":287088,"commit":"8bc83dd506556aeff40b450b5bfd44cb68626b64"}],"number":287150}
buildkite-pr-comment-->

paulinashakirova commented on src/platform/plugins/private/kibana_overview/public/components/manage_data/manage_data.tsx:
```diff
@@ -27,6 +36,7 @@ export const ManageData: FC<Props> = ({ addBasePath, features }) => {
   const {
     services: { application },
   } = useKibana<CoreStart>();
+  const minBreakpointM = useEuiMinBreakpoint('m');
```
WDYT about naming and using it like this?

paulinashakirova commented on src/platform/plugins/private/kibana_overview/public/components/overview/overview.tsx:
```diff
@@ -283,21 +289,37 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
-              ? 'kbnOverviewSupplements--hasNews'
-              : 'kbnOverviewSupplements--noNews'
+          className={`${
+            !newsFetchResult?.feedItems?.length ? 'kbnOverviewSupplements--noNews' : ''
```
I wonder if there is a better way to define styles here..

Dosant replied:
take a look at `classnames` util

paulinashakirova commented on src/platform/plugins/private/kibana_overview/public/components/overview/overview.tsx:
```diff
@@ -332,9 +364,29 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
+                        >
                           <EuiCard
-                            className={`kbnOverviewSolution ${id}`}
+                            className={`${id}`}
```
Having only an id feels a bit not intuitive - although this is what is needed...

eokoneyo replied:
We could alternatively, use a data attribute here, and target the data attribute; soemthing along the lines of 

```tsx
<EuiCard data-solution-id={id} ...
```
styles would then look somethijng along the lines of

```js
  '.kbnRedirectAppLinkImage': {
      '[data-solution-id="enterpriseSearch"]': {
        '.euiCard__image': {
          backgroundColor: euiTheme.colors.warning,
        },
      },
      '[data-solution-id="observability"]': {
        '.euiCard__image': {
          backgroundColor: euiTheme.colors.accent,
        },
      },
      '[data-solution-id="securitySolution"]': {
        '.euiCard__image': {
          backgroundColor: euiTheme.colors.accentSecondary,
        },
      },
    },
```

paulinashakirova replied:
Interesting suggestion I haven't thought of. Thank you

I would like to hear your thoughts behind this suggestion ☺️ 

From what I read - using `data-` is better when
- we need metadata for js logic 
- value is not related to styling

and using dynamic `className` when
- we need styling
- and we want to use with CSS selectors as it is somehow more performant (chatGPT - could be wrong or maybe it is really negligible)

My reasoning why in this particular case why would using `data-solution-id` is better is for the clarity it provides.

As an alternative, I could prepend the className (which I removed..., but maybe for the sake of clarity this is a good option to bring back)


I am open to try this out, and would like to learn from you.

Dosant replied:
I agree that just for styling using className is better and shows intent better. However, in this case I'd name it clearer: className="kbnOverviewSolution--{id}"

Commit:
improve implementation

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/287851)
* Commit: 81e98b0426330cf3a42ad73c6b4ee648f736d978

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/1d457e4b1bdc444f956f7f8feec6cc3415a4a605) | [after](https://github.com/elastic/kibana/commit/81e98b0426330cf3a42ad73c6b4ee648f736d978) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 93 | 80 | -13 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/1d457e4b1bdc444f956f7f8feec6cc3415a4a605) | [after](https://github.com/elastic/kibana/commit/81e98b0426330cf3a42ad73c6b4ee648f736d978) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 47.8KB | 37.9KB | -9.9KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/1d457e4b1bdc444f956f7f8feec6cc3415a4a605) | [after](https://github.com/elastic/kibana/commit/81e98b0426330cf3a42ad73c6b4ee648f736d978) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 5.7KB | 5.6KB | -47.0B |



### History
* :green_heart: [Build #287150](https://buildkite.com/elastic/kibana-pull-request/builds/287150) succeeded cb279e42f073d9071988b15c9cab750e5da42728
* :broken_heart: [Build #287088](https://buildkite.com/elastic/kibana-pull-request/builds/287088) failed 8bc83dd506556aeff40b450b5bfd44cb68626b64

cc @paulinashakirova

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287851","number":287851,"commit":"81e98b0426330cf3a42ad73c6b4ee648f736d978"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287150","number":287150,"commit":"cb279e42f073d9071988b15c9cab750e5da42728"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287088","number":287088,"commit":"8bc83dd506556aeff40b450b5bfd44cb68626b64"}],"number":287851}
buildkite-pr-comment-->

Commit:
Merge branch 'main' into scss-to-js-kubana-overview

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/288777)
* Commit: b5bec47d4dfdd8ed69f7932b60cd37b355952022

### Failed CI Steps
* [Jest Tests #1](https://buildkite.com/elastic/kibana-pull-request/builds/288777#0195edf6-c140-40c5-8943-67f78bdb08ee)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/288777#0195edf6-c140-40c5-8943-67f78bdb08ee) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/288777/jobs/0195edf6-c140-40c5-8943-67f78bdb08ee/artifacts/0195ee2c-5697-4540-aa68-a9afe84e1d58) Jest Tests #<span></span>1 / <KPIsSection /> should render all components

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/4970bb95ccb99b7cedfde51a3f4d56906607e28d) | [after](https://github.com/elastic/kibana/commit/b5bec47d4dfdd8ed69f7932b60cd37b355952022) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 93 | 80 | -13 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/4970bb95ccb99b7cedfde51a3f4d56906607e28d) | [after](https://github.com/elastic/kibana/commit/b5bec47d4dfdd8ed69f7932b60cd37b355952022) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 47.8KB | 37.9KB | -9.9KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/4970bb95ccb99b7cedfde51a3f4d56906607e28d) | [after](https://github.com/elastic/kibana/commit/b5bec47d4dfdd8ed69f7932b60cd37b355952022) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 5.7KB | 5.6KB | -47.0B |



### History
* :green_heart: [Build #287851](https://buildkite.com/elastic/kibana-pull-request/builds/287851) succeeded 81e98b0426330cf3a42ad73c6b4ee648f736d978
* :green_heart: [Build #287150](https://buildkite.com/elastic/kibana-pull-request/builds/287150) succeeded cb279e42f073d9071988b15c9cab750e5da42728
* :broken_heart: [Build #287088](https://buildkite.com/elastic/kibana-pull-request/builds/287088) failed 8bc83dd506556aeff40b450b5bfd44cb68626b64

cc @paulinashakirova

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288777","number":288777,"commit":"b5bec47d4dfdd8ed69f7932b60cd37b355952022"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287851","number":287851,"commit":"81e98b0426330cf3a42ad73c6b4ee648f736d978"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287150","number":287150,"commit":"cb279e42f073d9071988b15c9cab750e5da42728"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287088","number":287088,"commit":"8bc83dd506556aeff40b450b5bfd44cb68626b64"}],"number":288777}
buildkite-pr-comment-->

Dosant commented on src/platform/plugins/private/kibana_overview/public/components/overview/overview.tsx:
```diff
@@ -271,7 +269,15 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
           <>
             <EuiFlexGroup
-              className="kbnOverviewApps__group kbnOverviewApps__group--primary"
+              css={({ euiTheme }: UseEuiTheme) =>
```
WDYT of keeping LARGE styles outside of the component? 
Not necessarily a separate file, but at the top of the file under const styles = {}

Here is an example from Clint 

https://github.com/elastic/kibana/blob/a21b87fe65e853207b85fb1e097fe37511844c9a/src/core/packages/workspace/components/application/workspace_application.styles.ts


cc @tsullivan @eokoneyo

paulinashakirova replied:
Thank you for sharing this!
In some other files I was defining bigger chunks of styles outside of the component or at least at the root level - this is a great pattern indeed.
In this particular case I was hesitant and wanted to suggest this option, because (it is subjective, I am open to changes) the style property is the same "maxWidth", but it varies based on whether it is main app, remaining or a solution.
But since you mentioned it - I am going to move the styles to one place. ☺️
Now that I am looking at it - keeping in the same spot as in other places makes sense.

Commit:
refactor styles

Commit:
Merge branch 'main' into scss-to-js-kubana-overview

Commit:
refactor to fix ci

eokoneyo commented on src/platform/plugins/private/kibana_overview/public/components/overview/overview.tsx:
```diff
@@ -76,7 +79,11 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
-
+  const { euiTheme } = useEuiTheme();
+  const minBreakpointM = useEuiMinBreakpoint('m');
+  const className = classNames(
+    !newsFetchResult?.feedItems?.length && 'kbnOverviewSupplements--noNews'
+  );
```
we might rewrite this like so; 

```suggestion
classnames({
'kbnOverviewSupplements--noNews': !newsFetchResult?.feedItems?.length
})

```

Commit:
Update src/platform/plugins/private/kibana_overview/public/components/overview/overview.tsx

Co-authored-by: Eyo O. Eyo <7893459+eokoneyo@users.noreply.github.com>

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint --no-cache --fix'

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/290212)
* Commit: 4bd099c165d88e880c3699100880cda84d2d05a3

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/290212#01960054-f593-490c-b56f-8be86461b8f8)

### History
* :green_heart: [Build #290027](https://buildkite.com/elastic/kibana-pull-request/builds/290027) succeeded e1b5743b3e0def16082b52ae60db77c51e343edc
* :broken_heart: [Build #290005](https://buildkite.com/elastic/kibana-pull-request/builds/290005) failed 8b611cf50203f243f0d7b86e1f91f98fc1599d33
* :yellow_heart: [Build #288777](https://buildkite.com/elastic/kibana-pull-request/builds/288777) was flaky b5bec47d4dfdd8ed69f7932b60cd37b355952022
* :green_heart: [Build #287851](https://buildkite.com/elastic/kibana-pull-request/builds/287851) succeeded 81e98b0426330cf3a42ad73c6b4ee648f736d978
* :green_heart: [Build #287150](https://buildkite.com/elastic/kibana-pull-request/builds/287150) succeeded cb279e42f073d9071988b15c9cab750e5da42728
* :broken_heart: [Build #287088](https://buildkite.com/elastic/kibana-pull-request/builds/287088) failed 8bc83dd506556aeff40b450b5bfd44cb68626b64

cc @paulinashakirova

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290212","number":290212,"commit":"4bd099c165d88e880c3699100880cda84d2d05a3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290027","number":290027,"commit":"e1b5743b3e0def16082b52ae60db77c51e343edc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290005","number":290005,"commit":"8b611cf50203f243f0d7b86e1f91f98fc1599d33"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288777","number":288777,"commit":"b5bec47d4dfdd8ed69f7932b60cd37b355952022"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287851","number":287851,"commit":"81e98b0426330cf3a42ad73c6b4ee648f736d978"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287150","number":287150,"commit":"cb279e42f073d9071988b15c9cab750e5da42728"}],"number":290212}
buildkite-pr-comment-->

Commit:
fix errors after last commit on github

Commit:
Merge branch 'main' into scss-to-js-kubana-overview

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/290784)
* Commit: ab0c3310c939da2e5d02089f3a10b63642186c6c

### Failed CI Steps
* [FTR Configs #106](https://buildkite.com/elastic/kibana-pull-request/builds/290784#01961097-473b-4def-8320-333588e1e9cf)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/338aa07e0828fe5b7f2a5049c16399abe365777a) | [after](https://github.com/elastic/kibana/commit/ab0c3310c939da2e5d02089f3a10b63642186c6c) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 93 | 80 | -13 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/338aa07e0828fe5b7f2a5049c16399abe365777a) | [after](https://github.com/elastic/kibana/commit/ab0c3310c939da2e5d02089f3a10b63642186c6c) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 47.8KB | 38.1KB | -9.7KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/338aa07e0828fe5b7f2a5049c16399abe365777a) | [after](https://github.com/elastic/kibana/commit/ab0c3310c939da2e5d02089f3a10b63642186c6c) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 5.7KB | 5.6KB | -47.0B |



### History
* :broken_heart: [Build #290212](https://buildkite.com/elastic/kibana-pull-request/builds/290212) failed 4bd099c165d88e880c3699100880cda84d2d05a3
* :green_heart: [Build #290027](https://buildkite.com/elastic/kibana-pull-request/builds/290027) succeeded e1b5743b3e0def16082b52ae60db77c51e343edc
* :broken_heart: [Build #290005](https://buildkite.com/elastic/kibana-pull-request/builds/290005) failed 8b611cf50203f243f0d7b86e1f91f98fc1599d33
* :yellow_heart: [Build #288777](https://buildkite.com/elastic/kibana-pull-request/builds/288777) was flaky b5bec47d4dfdd8ed69f7932b60cd37b355952022
* :green_heart: [Build #287851](https://buildkite.com/elastic/kibana-pull-request/builds/287851) succeeded 81e98b0426330cf3a42ad73c6b4ee648f736d978
* :green_heart: [Build #287150](https://buildkite.com/elastic/kibana-pull-request/builds/287150) succeeded cb279e42f073d9071988b15c9cab750e5da42728

cc @paulinashakirova

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290784","number":290784,"commit":"ab0c3310c939da2e5d02089f3a10b63642186c6c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290212","number":290212,"commit":"4bd099c165d88e880c3699100880cda84d2d05a3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290027","number":290027,"commit":"e1b5743b3e0def16082b52ae60db77c51e343edc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290005","number":290005,"commit":"8b611cf50203f243f0d7b86e1f91f98fc1599d33"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288777","number":288777,"commit":"b5bec47d4dfdd8ed69f7932b60cd37b355952022"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287851","number":287851,"commit":"81e98b0426330cf3a42ad73c6b4ee648f736d978"}],"number":290784}
buildkite-pr-comment-->

Commit:
Merge branch 'main' into scss-to-js-kubana-overview

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/293629)
* Commit: 1586d26aa9967a914e7e52c18995318a0cc1d6be

### Failed CI Steps
* [Jest Tests #11](https://buildkite.com/elastic/kibana-pull-request/builds/293629#01963e48-7a08-4c54-a1ca-05b6a3027f38)
* [Jest Tests #11](https://buildkite.com/elastic/kibana-pull-request/builds/293629#01963e70-8f27-42ad-8085-10b02f2f940d)
* [FTR Configs #2](https://buildkite.com/elastic/kibana-pull-request/builds/293629#01963e48-7a16-4074-ae92-b624a97934ab)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293629#01963e48-7a08-4c54-a1ca-05b6a3027f38) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293629/jobs/01963e48-7a08-4c54-a1ca-05b6a3027f38/artifacts/01963e70-64b7-4b7f-af3b-74c1b89dca86) Jest Tests #<span></span>11 / FeedItem render
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293629#01963e70-8f27-42ad-8085-10b02f2f940d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293629/jobs/01963e70-8f27-42ad-8085-10b02f2f940d/artifacts/01963e76-03ed-45c4-ae00-7a7861790b7a) Jest Tests #<span></span>11 / FeedItem render
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293629#01963e48-7a08-4c54-a1ca-05b6a3027f38) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293629/jobs/01963e48-7a08-4c54-a1ca-05b6a3027f38/artifacts/01963e70-64b7-4930-b95b-c8a92e50a386) Jest Tests #<span></span>11 / ManageData render
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293629#01963e70-8f27-42ad-8085-10b02f2f940d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293629/jobs/01963e70-8f27-42ad-8085-10b02f2f940d/artifacts/01963e76-03ed-4081-85d6-ec0c1b3cdb93) Jest Tests #<span></span>11 / ManageData render

### History
* :yellow_heart: [Build #290784](https://buildkite.com/elastic/kibana-pull-request/builds/290784) was flaky ab0c3310c939da2e5d02089f3a10b63642186c6c
* :broken_heart: [Build #290212](https://buildkite.com/elastic/kibana-pull-request/builds/290212) failed 4bd099c165d88e880c3699100880cda84d2d05a3
* :green_heart: [Build #290027](https://buildkite.com/elastic/kibana-pull-request/builds/290027) succeeded e1b5743b3e0def16082b52ae60db77c51e343edc
* :broken_heart: [Build #290005](https://buildkite.com/elastic/kibana-pull-request/builds/290005) failed 8b611cf50203f243f0d7b86e1f91f98fc1599d33
* :yellow_heart: [Build #288777](https://buildkite.com/elastic/kibana-pull-request/builds/288777) was flaky b5bec47d4dfdd8ed69f7932b60cd37b355952022
* :green_heart: [Build #287851](https://buildkite.com/elastic/kibana-pull-request/builds/287851) succeeded 81e98b0426330cf3a42ad73c6b4ee648f736d978

cc @paulinashakirova

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293629","number":293629,"commit":"1586d26aa9967a914e7e52c18995318a0cc1d6be"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290784","number":290784,"commit":"ab0c3310c939da2e5d02089f3a10b63642186c6c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290212","number":290212,"commit":"4bd099c165d88e880c3699100880cda84d2d05a3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290027","number":290027,"commit":"e1b5743b3e0def16082b52ae60db77c51e343edc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290005","number":290005,"commit":"8b611cf50203f243f0d7b86e1f91f98fc1599d33"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288777","number":288777,"commit":"b5bec47d4dfdd8ed69f7932b60cd37b355952022"}],"number":293629}
buildkite-pr-comment-->

alexwizp committed:
fix: [Analytics:Graph page]Dialog modal missing title from announcement (#217827)

Closes: #214760

**Description**
Dialog modal, flyout, field visible title should be announced for the
users, especially using assistive technology to know what dialog modal,
flyout opened, what field is active and what is needed to enter in it.

**Changes made:**

1. Set correct value for` aria-labelledby` attr.

semd committed:
[Security Solution] Allow disabling experimental features via config (#217363)

## Summary

This PR adds support for disabling experimental features using the
existing `xpack.securitySolution.enableExperimental` configuration.

This solves the problem of not being able to disable a feature by config
once the feature has been enabled by default.

### The Challenge 

When we start developing a feature under an experimental flag we always
follow the same steps:

1 - Create the experimental flag disabled by default + enable it via
config for testing
2 - Implement the feature
3 - Enable the experimental flag by default when we want to release the
feature.
4 - Deployments can disable the feature via config (as a safety
measure).
5 - Remove the experimental flag after some time.

We start by creating the flag disabled by default while we implement it.
In `experimental_features.ts`:
```ts
export const allowedExperimentalValues = Object.freeze({
  myFeatureEnabled: false,
  [...]
```
And enable it via config with:
```yml
xpack.securitySolution.enableExperimental:
  - myFeatureEnabled
```

Once the implementation is done and the experimental flag can be enabled
by default, we have to do a trick:
Since the `xpack.securitySolution.enableExperimental` config can only
turn flags to _true_, instead of setting `myFeatureEnabled: true`, what
we have to do is rename the flag to `myFeatureDisabled` and keep the
value as _false_:

```ts
export const allowedExperimentalValues = Object.freeze({
  myFeatureDisabled: false,
  [...]
```
Then we also need to do a code refactor to update all the places in the
code where the flag was checked: `if (myFeatureEnabled)` -> `if
(!myFeatureDisabled)`

This way, we have the option of disabling the feature via config (in
case something goes wrong):
```yml
xpack.securitySolution.enableExperimental:
  - myFeatureDisabled
```

### A solution

This PR introduces the possibility to turn a flag to _false_ using the
same `xpack.securitySolution.enableExperimental` config. This was
preferable to introducing a new config since this one is already
whitelisted in Cloud UI, can be easily overritten in deployments, and
also because people are used to it.

With these changes, the first two steps would be the same, with the
difference that we won't need to have the _Enabled_ or _Disabled_ word
at the end of the flag name. It could be just the feature name, in
`experimental_features.ts`:
```ts
export const allowedExperimentalValues = Object.freeze({
  myFeature: false,
  [...]
```

And when we need to enable the feature by default, we can just turn it
to `true`:
```ts
export const allowedExperimentalValues = Object.freeze({
  myFeature: true,
  [...]
```
No tedious refactor or confusing naming would be required. 

Then, in case we need to disable the feature in a production deployment
for some reason, we could just do this via config :
```yml
xpack.securitySolution.enableExperimental:
  - disable:myFeature
```

---------

Co-authored-by: Elastic Machine <elasticmachine@users.noreply.github.com>

Commit:
fix tests

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

Commit:
remove unrelated files +fix className

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/293739)
* Commit: f60a2573696141c7efe75e9db10883f05005fd1f

### Failed CI Steps
* [FTR Configs #67](https://buildkite.com/elastic/kibana-pull-request/builds/293739#01963ed2-de06-4814-86a3-75eb03b5e167)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293739#01963ed2-de06-4814-86a3-75eb03b5e167) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293739/jobs/01963ed2-de06-4814-86a3-75eb03b5e167/artifacts/01963ef4-7039-4746-a7b5-b617d3455d6d) FTR Configs #<span></span>67 / management index pattern filter "after each" hook for "should filter indexed fields by type"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293739#01963ed2-de06-4814-86a3-75eb03b5e167) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293739/jobs/01963ed2-de06-4814-86a3-75eb03b5e167/artifacts/01963ef4-7039-4823-aec8-8172a4fa4fd3) FTR Configs #<span></span>67 / management index pattern filter "before each" hook for "should filter indexed fields by type"

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/19feb59109b2cbc53667042e04ceba4f4e92c901) | [after](https://github.com/elastic/kibana/commit/f60a2573696141c7efe75e9db10883f05005fd1f) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 93 | 80 | -13 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/19feb59109b2cbc53667042e04ceba4f4e92c901) | [after](https://github.com/elastic/kibana/commit/f60a2573696141c7efe75e9db10883f05005fd1f) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 47.8KB | 38.0KB | -9.7KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/19feb59109b2cbc53667042e04ceba4f4e92c901) | [after](https://github.com/elastic/kibana/commit/f60a2573696141c7efe75e9db10883f05005fd1f) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 5.7KB | 5.6KB | -47.0B |



### History
* :broken_heart: [Build #293629](https://buildkite.com/elastic/kibana-pull-request/builds/293629) failed 1586d26aa9967a914e7e52c18995318a0cc1d6be
* :yellow_heart: [Build #290784](https://buildkite.com/elastic/kibana-pull-request/builds/290784) was flaky ab0c3310c939da2e5d02089f3a10b63642186c6c
* :broken_heart: [Build #290212](https://buildkite.com/elastic/kibana-pull-request/builds/290212) failed 4bd099c165d88e880c3699100880cda84d2d05a3
* :green_heart: [Build #290027](https://buildkite.com/elastic/kibana-pull-request/builds/290027) succeeded e1b5743b3e0def16082b52ae60db77c51e343edc
* :broken_heart: [Build #290005](https://buildkite.com/elastic/kibana-pull-request/builds/290005) failed 8b611cf50203f243f0d7b86e1f91f98fc1599d33

cc @paulinashakirova

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293739","number":293739,"commit":"f60a2573696141c7efe75e9db10883f05005fd1f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293629","number":293629,"commit":"1586d26aa9967a914e7e52c18995318a0cc1d6be"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290784","number":290784,"commit":"ab0c3310c939da2e5d02089f3a10b63642186c6c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290212","number":290212,"commit":"4bd099c165d88e880c3699100880cda84d2d05a3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290027","number":290027,"commit":"e1b5743b3e0def16082b52ae60db77c51e343edc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290005","number":290005,"commit":"8b611cf50203f243f0d7b86e1f91f98fc1599d33"}],"number":293739}
buildkite-pr-comment-->

Dosant commented on src/platform/test/functional/apps/kibana_overview/_solutions.ts:
```diff
@@ -39,7 +39,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     it('contains Security and Observability solutions', async () => {
-      const solutionCards = await find.allByCssSelector('.kbnOverviewMore__item');
+      const solutionCards = await find.allByCssSelector('[data-test-subj="kbnOverviewItem"]');
```
nit: await testSubjects.findAll('kbnOverviewItem')

Dosant commented on src/platform/plugins/private/kibana_overview/tsconfig.json:
```diff
@@ -6,6 +6,7 @@
   "include": [
     "public/**/*",
     "common/**/*",
+    "../../../../../typings/emotion.d.ts"
```
@eokoneyo should be an expert in this! Is it really needed? If so, I think something else might be wrong.

Dosant commented on src/platform/plugins/private/kibana_overview/public/components/overview/overview.tsx:
```diff
@@ -76,6 +79,12 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
+  const { euiTheme } = useEuiTheme();
+  const minBreakpointM = useEuiMinBreakpoint('m');
+
+  const className = classNames({
```
nit? 

```suggestion
  const newsContainerClassName = classNames({
```

or just make it inline?

Dosant reviewed (commented):
One visual difference I noticed is that the cards under "Do More with Elastic" are too large.

![Screenshot 2025-04-17 at 12 39 42](https://github.com/user-attachments/assets/ce40cc7a-8607-4b8c-9b26-67d8f0e3d3bb)

Commit:
Merge branch 'main' into scss-to-js-kubana-overview

Commit:
resolve comments

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/294241)
* Commit: b53ff1956bd0c391b6d6d8fe81d5de6f9d4e063c

### Failed CI Steps
* [FTR Configs #30](https://buildkite.com/elastic/kibana-pull-request/builds/294241#019644b6-a50a-4dce-b75b-07e73758b501)
* [FTR Configs #30](https://buildkite.com/elastic/kibana-pull-request/builds/294241#019644da-54d7-4ef1-8869-3cb4d814b321)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294241#019644b6-a50a-4dce-b75b-07e73758b501) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294241/jobs/019644b6-a50a-4dce-b75b-07e73758b501/artifacts/019644da-45d3-410b-8602-e7faa6eeb619) FTR Configs #<span></span>30 / kibana overview app overview page - solutions contains Security and Observability solutions
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294241#019644da-54d7-4ef1-8869-3cb4d814b321) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294241/jobs/019644da-54d7-4ef1-8869-3cb4d814b321/artifacts/019644e0-4292-428b-80c4-31e54b650dcd) FTR Configs #<span></span>30 / kibana overview app overview page - solutions contains Security and Observability solutions

### History
* :green_heart: [Build #294195](https://buildkite.com/elastic/kibana-pull-request/builds/294195) succeeded 696e674edc99fad4841ae44a4c627f67e3ffe288
* :yellow_heart: [Build #293739](https://buildkite.com/elastic/kibana-pull-request/builds/293739) was flaky f60a2573696141c7efe75e9db10883f05005fd1f
* :broken_heart: [Build #293629](https://buildkite.com/elastic/kibana-pull-request/builds/293629) failed 1586d26aa9967a914e7e52c18995318a0cc1d6be
* :yellow_heart: [Build #290784](https://buildkite.com/elastic/kibana-pull-request/builds/290784) was flaky ab0c3310c939da2e5d02089f3a10b63642186c6c
* :broken_heart: [Build #290212](https://buildkite.com/elastic/kibana-pull-request/builds/290212) failed 4bd099c165d88e880c3699100880cda84d2d05a3
* :green_heart: [Build #290027](https://buildkite.com/elastic/kibana-pull-request/builds/290027) succeeded e1b5743b3e0def16082b52ae60db77c51e343edc

cc @paulinashakirova

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294241","number":294241,"commit":"b53ff1956bd0c391b6d6d8fe81d5de6f9d4e063c"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294195","number":294195,"commit":"696e674edc99fad4841ae44a4c627f67e3ffe288"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293739","number":293739,"commit":"f60a2573696141c7efe75e9db10883f05005fd1f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293629","number":293629,"commit":"1586d26aa9967a914e7e52c18995318a0cc1d6be"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290784","number":290784,"commit":"ab0c3310c939da2e5d02089f3a10b63642186c6c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290212","number":290212,"commit":"4bd099c165d88e880c3699100880cda84d2d05a3"}],"number":294241}
buildkite-pr-comment-->

Commit:
use in ftr testid not classname for overview

Commit:
Merge branch 'main' into scss-to-js-kubana-overview

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/294811)
* Commit: 3ee407e6c0ae60c1097910adb2f763b4e09dfbd6

### Failed CI Steps
* [Jest Tests #11](https://buildkite.com/elastic/kibana-pull-request/builds/294811#01965cef-6a2a-451a-9fc6-9dfa07588c0f)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294811#01965cef-6a2a-451a-9fc6-9dfa07588c0f) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294811/jobs/01965cef-6a2a-451a-9fc6-9dfa07588c0f/artifacts/01965d17-1777-43c0-80ab-c6f09305079b) Jest Tests #<span></span>11 / TemplatesList renders edit button

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/fe31818a3b8a7630d3d9887ac13cdc7932298ac1) | [after](https://github.com/elastic/kibana/commit/3ee407e6c0ae60c1097910adb2f763b4e09dfbd6) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 93 | 80 | -13 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/fe31818a3b8a7630d3d9887ac13cdc7932298ac1) | [after](https://github.com/elastic/kibana/commit/3ee407e6c0ae60c1097910adb2f763b4e09dfbd6) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 47.8KB | 38.1KB | -9.7KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/fe31818a3b8a7630d3d9887ac13cdc7932298ac1) | [after](https://github.com/elastic/kibana/commit/3ee407e6c0ae60c1097910adb2f763b4e09dfbd6) | diff |
| --- | --- | --- | --- |
| `kibanaOverview` | 5.7KB | 5.6KB | -47.0B |



### History
* :broken_heart: [Build #294241](https://buildkite.com/elastic/kibana-pull-request/builds/294241) failed b53ff1956bd0c391b6d6d8fe81d5de6f9d4e063c
* :green_heart: [Build #294195](https://buildkite.com/elastic/kibana-pull-request/builds/294195) succeeded 696e674edc99fad4841ae44a4c627f67e3ffe288
* :yellow_heart: [Build #293739](https://buildkite.com/elastic/kibana-pull-request/builds/293739) was flaky f60a2573696141c7efe75e9db10883f05005fd1f
* :broken_heart: [Build #293629](https://buildkite.com/elastic/kibana-pull-request/builds/293629) failed 1586d26aa9967a914e7e52c18995318a0cc1d6be
* :yellow_heart: [Build #290784](https://buildkite.com/elastic/kibana-pull-request/builds/290784) was flaky ab0c3310c939da2e5d02089f3a10b63642186c6c

cc @paulinashakirova

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294811","number":294811,"commit":"3ee407e6c0ae60c1097910adb2f763b4e09dfbd6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294241","number":294241,"commit":"b53ff1956bd0c391b6d6d8fe81d5de6f9d4e063c"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294195","number":294195,"commit":"696e674edc99fad4841ae44a4c627f67e3ffe288"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293739","number":293739,"commit":"f60a2573696141c7efe75e9db10883f05005fd1f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293629","number":293629,"commit":"1586d26aa9967a914e7e52c18995318a0cc1d6be"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/290784","number":290784,"commit":"ab0c3310c939da2e5d02089f3a10b63642186c6c"}],"number":294811}
buildkite-pr-comment-->
